packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerCellExample.shorthand.tsx

```diff
@@ -0,0 +1,22 @@
+import { Datepicker, Tooltip } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const DatepickerCellExample = () => {
+  return (
+    <Datepicker
+      today={new Date(2020, 8, 12, 0, 0, 0, 0)}
+      calendar={{
+        calendarCell: {
+          children: (ComponentType, props) =>
+            props.isToday ? (
```

Comment:
Is it really `isToday`? We don't use `is*` prefix in prop names...

Reply:
Ooops, will fix it with @rymeskar in a separate PR :D